# Galaxy training course for metagenomics analysis
Date: 18th -22nd April (tentative)
Time: 9:30 am – 4:00pm
Venue: Vivarium
Capacity: 20 participants
Target: Scientists working with metagenomics data
Website: The resources used in this training are – or will be – available on this GitHub Repo.

## Introduction
Metagenomics is a field that allows scientists to analyze the genomes of uncultured microbes. Information about microorganisms in the environment can be extracted using two main approaches in metagenomics: amplicon sequencing, which sequences only the rRNA or ribosomal DNA of organisms, and shotgun sequencing, which sequences the entire genomes of the microbes in the environment. Amplicon sequencing is a method for detecting genetic variation in specific genomic areas that is highly focused. Amplicon sequencing is a term used in the metagenomics area to describe the acquisition and sequencing of rRNA data in a sample. For bacteria and archea, it's 16S, and for eukaryotes, it's 18S. Full genomes of microorganisms in the environment are sequenced using shotgun data (not only 16S or 18S).
The workshop will focus on 16S rRNA gene analysis in Galaxy because it is the most prevalent method for studying bacterial phylogeny and genus/species categorization that uses housekeeping genes. The 16S ribosomal RNA gene codes for the RNA component of the 30S subunit of the bacterial ribosome. It's found in almost every bacterial species (E. Stackebrandt and B. M. Goebel, 1994). The 16S rRNA gene is found in one to several copies in different bacterial species. 16S rRNA gene sequencing is used to distinguish between closely related bacterial species and to identify bacteria at the species level. This approach is used by many clinical laboratories to identify unknown pathogenic pathogens.
The training targets all researchers and students who generate metagenomics data, and will focus on how to analyze the metagenomics data. This training will set a foundation for data curation, and ultimately metagenomic analysis, which is one of the strategic focus areas of icipe.

## Learning Outcomes
The workshop participant should acquire the following skills;
1.	Be able to understand the importance of Bioinformatics in research.
2.	Be able to run jobs on the High-Performance Computing environment.
3.	Be able to understand the different omics field.
4.	Be able to run analysis on Galaxy.
5.	Be able to understand how to run their 16s data on Galaxy.

## Instructors
•	Dr. Caleb Kibet
•	Andrew Espira
•	Ruth Nanjala
•	Pauline Karega
•	Nehemiah Ongeso
•	Dr. Jandouwe Villinger

## Who should attend?
This training targets students and researchers within icipe who are working with metagenomics data. Come with your data.
Training Topics
•	Introduction to Bioinformatics
•	Introduction to genomics 
•	Introduction to HPC and best practices for using the HPC
•	Introduction to Galaxy
•	Analysing 16s data on Galaxy
